package/MDAnalysis/lib/qcprot.pyx

```diff
@@ -240,6 +246,41 @@ def InnerProduct(np.ndarray[np.float64_t,ndim=1] A,
 
 @cython.boundscheck(False)
 @cython.wraparound(False)
+def CalcRMSDRotationalMatrix(np.ndarray[np.float64_t,ndim=2] ref,
```

Comment:
Did you look into using [Memoryviews](http://docs.cython.org/en/latest/src/userguide/memoryviews.html). They can give some additional speedup since they are really raw pointers. The `np.ndarray` objects are a little bit more.